library/spdm_requester_lib/libspdm_req_get_capabilities.c

```diff
@@ -137,6 +153,22 @@ static bool validate_responder_capability(uint32_t capabilities_flag, uint8_t ve
         }
     }
 
+    /* Checks specific to 1.3. */
+    if (version == SPDM_MESSAGE_VERSION_13) {
+        /* Illegal to return reserved values. */
+        if ((ep_info_cap == 3) || (multi_key_cap == 3)) {
+            return false;
+        }
+        /* check multi-key */
+        if ((multi_key_cap != 0) && (get_key_pair_info_cap == 0)) {
+            return false;
+        }
+        if ((multi_key_cap == 0) &&
+            ((get_key_pair_info_cap == 1) || (set_key_pair_info_cap == 1))) {
```

Comment:
Filed https://github.com/DMTF/SPDM-WG/issues/3270 for this one.

Reply:
Will remove this check.